Logger/FileLogger.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Logger
+{
+    public class FileLogger: BaseLogger
+    {
+        private string? ClassName { get; set; }
+        private string? FilePath { get; set; }
```

Comment:
```suggestion
        private string? FilePath { get;  }
```
You can make this readonly